src/build.ps1

```diff
@@ -0,0 +1,49 @@
+[cmdletbinding()]
```

Comment:
I think this should reside at the root of the repo.

Reply:
Agree.  Will update.